src/openrct2/paint/tile_element/Paint.TileElement.h

```diff
@@ -11,6 +11,7 @@
 
 #include "../../common.h"
 #include "../../world/Map.h"
+#include "../Paint.SessionFlags.h"
```

Comment:
do you really need it in here? Its best to really try as hard as possible to put headers in headers.

Reply:
I suppose not.